client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -42,6 +52,28 @@ const validate = ( values, props ) => {
 
 	if ( ! values.price ) {
 		errors.price = translate( 'Price can not be empty.' );
+	} else if ( parseFloat( values.price ) === NaN ) {
+		errors.price = translate( 'Price is invalid.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Invalid price' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).